src/renderer/getDiForUnitTesting.tsx

```diff
@@ -31,6 +33,12 @@ export const getDiForUnitTesting = ({ doGeneralOverrides } = { doGeneralOverride
   if (doGeneralOverrides) {
     di.override(getValueFromRegisteredChannelInjectable, () => () => undefined);
 
+    di.override(readDirInjectable, () => () => Promise.resolve([]));
+
+    di.override(readFileInjectable, () => () => {
+      throw new Error("Tried to read a file from file system without specifying explicit overrride");
```

Comment:
Typo: `overrride` -> `override.`

Reply:
I think that was so that you roll the r's when you say it :)